ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/binutils/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+############
+# binutils #
+############
+
+# gcc-7.3 requires binutils v2.24 or newer
+# Note the installation on CentOS x86_64 is done via Developer Toolset 7 from
+# Software Collections (see the Common role for CentOS).
+
+# Note:
+# In order to use the binutils 2.31 tools (ld, as, ar) add:
+# /usr/local/binutils/bin to PATH
+# /usr/local/binutils/lib to LD_LIBRARY_PATH
+
+- name: Get the binutils version on RHEL/CentOS
+  shell: (/usr/local/binutils/bin/ld -v 2>/dev/null || ld -v 2>/dev/null ) | sed -n 's/^.*\( [0-9]\.[0-9][0-9]\).*/\1/p'| awk '{print $1}'
```

Comment:
This appears to ignore the possibility that there will be a suitable version installed in /usr/bin (RHEL7)

Reply:
`... || ld -v` should take care of the one installed in /usr/bin.

Reply:
Fair point :-)